src/test/kotlin/chapter8/solutions/ex4/listing.kt

```diff
@@ -11,5 +12,12 @@ data class Gen<A>(val sample: State<RNG, A>) {
             Gen(State { rng: RNG -> nonNegativeInt(rng) }
                 .map { start + (it % (stopExclusive - start)) })
         //end::init[]
+
+        //tag::init2[]
+        fun chooseEvenly(start: Int, stopExclusive: Int): Gen<Int> =
```

Comment:
Should we perhaps call this `chooseUnbiased` instead?

Reply:
Done!